### PR TITLE
[Widget Mode] Add unit and eyes tests

### PR DESCRIPTION
**Pull request by jmkulwik**
When widget mode was implemented, it did not include tests because it was being built for a rapidly approaching pilot. These tests are being added retroactively. For a diff of the entire widget mode feature, see this PR: https://github.com/code-dot-org/code-dot-org/pull/30189

The new allthethings level will look like this:
image

### CODE REVIEW

File: dashboard/config/scripts/allthethings.script [hunk 1]
```diff
@@ -156,10 +157,10 @@ level 'Test longassessment levelgroup split new', assessment: true
 stage 'Star wars'
 skin 'hoc2015'
 video_key_for_next_level 'starwars_events'
-level_concept_difficulty '{}'
+level_concept_difficulty '{"sequencing":1}'
```
Comment (ajpal) [inline dashboard/config/scripts/allthethings.script]: What does this change do? (And below)

File: dashboard/test/ui/features/applab/eyes2.feature [hunk 2]
```diff
@@ -35,6 +35,13 @@ Scenario: Applab embedded level
   And I see no difference for "embedded level"
   Then I close my eyes
 
+Scenario: Applab widget mode
+  When I open my eyes to test "Applab widget mode"
+  And I am on "http://studio.code.org/s/allthethings/stage/18/puzzle/22"
+  And I wait until element "#reset" is visible
```
Comment (ajpal) [inline dashboard/test/ui/features/applab/eyes2.feature]: I thought widget mode didn't show the reset button?

Comment (jmkulwik) [reply dashboard/test/ui/features/applab/eyes2.feature]: Ahh sorry, this is a button within the widget app with id `reset`. The reset button has id `resetButton`. I'll see what I can do to edit the widget app a bit to make this more clear.

File: dashboard/config/scripts/allthethings.script (hunk 1 above)
Comment (jmkulwik) [reply dashboard/config/scripts/allthethings.script]: Oh weird. Must be some sort of merge issue. I'll look into what happened here.

Comment (jmkulwik) [reply dashboard/config/scripts/allthethings.script]: Removed

File: dashboard/test/ui/features/applab/eyes2.feature (hunk 2 above)
Comment (jmkulwik) [reply dashboard/test/ui/features/applab/eyes2.feature]: Fixed!

**Review by ajpal — APPROVED** [review]

**Pull request merged**